### ISSUE TITLE
Support for reqwest 0.12 ?

### ISSUE THREAD

**Issue by sbfaulkner** [graphql-rust/graphql-client#481]
The graphql_client optional dependency for reqwest-create seems locked to 0.11

https://github.com/graphql-rust/graphql-client/blob/main/graphql_client/Cargo.toml#L23

What's needed to support use with 0.12?

I currently have issues because of conflicting versions if I try to use 0.12 in a project with reqwest and graphql_client

**Comment by sahra-karakoc** [comment]
Any progress/plans on this?

**Comment by tomhoule** [comment]
Nobody is working on it, but I could review a PR. We need different feature flags for different versions of reqwest.

**Comment by jpopesculian** [comment]
Ahh, I opened a PR #490 but I'll add feature flags

**Comment by Sytten** [comment]
PR https://github.com/graphql-rust/graphql-client/pull/494 is better, should be able to merge as is and release a patch version

**Comment by sbfaulkner** [comment]
closed by #499 I think?

**Comment by tomhoule** [comment]
Indeed, thanks.

**Comment by uncenter** [comment]
Would it be possible to make a release including this change?

**Comment by tomhoule** [comment]
Yes, there'll be a release as soon as I or another maintainer can free up some time.
